graphql-schemas/schema.yml

```diff
@@ -2309,7 +2303,6 @@
   - { name: sentry_teams_v1, type: SentryTeam_v1, isList: true, datafileSchema: /dependencies/sentry-team-1.yml }
   - { name: sentry_instances_v1, type: SentryInstance_v1, isList: true, datafileSchema: /dependencies/sentry-instance-1.yml }
   - { name: app_interface_sql_queries_v1, type: AppInterfaceSqlQuery_v1, isList: true, datafileSchema: /app-interface/app-interface-sql-query-1.yml }
-  - { name: saas_files_v1, type: SaasFile_v1, isList: true, datafileSchema: /app-sre/saas-file-1.yml }
```

Comment:
what about the actual `SaasFile_v1` item?

Reply:
Good catch. I also saw that `Role_v1` wrongly references `SaasFile_v1`, even though since some time now in app-interface the roles use `SaasFile_v2` files.